ecs-cli/modules/utils/regcredio/creds_readers_test.go

```diff
@@ -225,3 +226,38 @@ func TestReadCredsOutput_ErrorFileNotFound(t *testing.T) {
 	_, err := ReadCredsOutput(fakeFileName)
 	assert.Error(t, err, "Expected error on missing file")
 }
+
+func TestFindLatestRegCredsOutputFile(t *testing.T) {
+	testCases := []struct {
+		inputFileNames []string
+		expectedLatest string
+	}{
+		{[]string{"ecs-registry-creds_20171117T125102Z.yml", "ecs-registry-creds_20181012T215145Z.yml", "ecs-registry-creds_20181017T125102Z.yml"}, "ecs-registry-creds_20181017T125102Z.yml"},
```

Comment:
:+1: on table based tests -- it might be worth adding a `description` in the testCases struct field to make it even clearer what each test case is doing.